src/time_integration/paired_explicit_runge_kutta/methods_PERK2.jl

```diff
@@ -0,0 +1,392 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+using DelimitedFiles: readdlm
+using LinearAlgebra: eigvals
+using .PolynomialOptimizer
+
+import .PolynomialOptimizer: filter_eigvals, undo_normalization!, bisection
+
+@muladd begin
+#! format: noindent
+
+# Abstract base type for both single/standalone and multi-level 
+# PERK (Paired-Explicit Runge-Kutta) time integration schemes
+abstract type PERK end
+# Abstract base type for single/standalone PERK time integration schemes
+abstract type PERKSingle <: PERK end
+
+function compute_a_coeffs(num_stage_evals, bc_factors, mon_coeffs)
```

Comment:
My comments about abbreviated variable names (made either above or below) also apply to all functions in this file